builtins/xrootd/CMakeLists.txt

```diff
@@ -76,7 +76,9 @@ set(XROOTD_LIBRARY ${XROOTD_PREFIX}/${XROOTD_LIBDIR}/${XROOTD_LIBNAME} CACHE INT
 set(XROOTD_LIBRARIES ${XROOTD_LIBRARIES} CACHE INTERNAL "" FORCE)
 
 add_library(xrootd INTERFACE)
-target_include_directories(xrootd INTERFACE $<BUILD_INTERFACE:${XROOTD_INCLUDE_DIR}>)
+foreach(dir ${XROOT_INCLUDE_DIR})
```

Comment:
Is it `${XROOT_INCLUDE_DIR}` or `${XROOTD_INCLUDE_DIR}`?

Reply:
It is the later :(